config/initializers/new_framework_defaults_7_0.rb

```diff
@@ -0,0 +1,117 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
this file isn't needed, you are already loading the rails 7.0 defaults here https://github.com/rootstrap/rails_api_base/pull/357/files#diff-c1fd91cb1911a0512578b99f657554526f3e1421decdb9e908712beab57e10f9R25